pylint/lint.py

```diff
@@ -1317,6 +1255,78 @@ def _report_evaluation(self):
             self.reporter.display_reports(sect)
 
 
+def check_parallel(linter, jobs, files):
+    """Use the given linter to lint the files with given amount of workers (jobs)
+    """
+    # The reporter does not need to be passed to worker processess, i.e. the reporter does
+    # not need to be pickleable
+    original_reporter = linter.reporter
+    linter.reporter = None
+
+    # The linter is inherited by all the pool's workers, i.e. the linter
+    # is identical to the linter object here. This is requred so that
+    # a custom PyLinter object (inherited from PyLinter) can be used.
+    # See https://github.com/PyCQA/prospector/issues/320
```

Comment:
I don't think a link to prospector's issue tracker adds any value here, can we remove and expand the comment if needed more context instead?

Reply:
done